netdisco/discoverables/webos_tv.py

```diff
@@ -13,7 +13,21 @@ def info_from_entry(self, entry):
 
     def get_entries(self):
         """Get all the LG WebOS TV device uPnP entries."""
-        return self.find_by_device_description({
-            "deviceType": "urn:dial-multiscreen-org:device:dial:1",
-            "friendlyName": "[LG] webOS TV"
-        })
+
+        # Lookup for webos3
+        webos_entry = self.find_by_device_description(
+            {
+                "deviceType": "urn:schemas-upnp-org:device:MediaRenderer:1",
+                "manufacturer": "LG Electronics.",
+                "modelName": "LG TV"
+            }
+        )
+
+        if not len(webos_entry):
```

Comment:
The thing here is that you don't want to search for one thing if you didn't find the other. That results in unpredictable behavior for the user. Because one time it might find 1 TV, but if that TV is off it will use the other search and find 2 other ones. That doesn't work.